theme/sass/components/_global.scss

```diff
@@ -43,4 +43,19 @@ body {
 .responsive-image {
     width: 100%;
     height: auto;
-}
\ No newline at end of file
+}
+
+.clearfix:after {
+    content: " ";
```

Comment:
Add the :after insider the clearfix class and above it add `clear: both;` so that the class also has clear both on it.

Reply:
done